aviary/subsystems/propulsion/engine_scaling.py

```diff
@@ -316,36 +282,6 @@ def compute_partials(self, inputs, J):
             scale_factor = engine_scale_factor
             deriv_factor = 1.0
 
-        # J[Dynamic.THRUST, 'thrust_net_unscaled'] = np.tile(
```

Comment:
Engine scaling used to be vectorized across all engines (so variables were 2d vectors of engine types and num_nodes), but this doesn't work for custom engine models that don't use this component, so I'm removing all the old code that supported that type of vectorization. It was kept there just in case it would be useful someday, but that doesn't look like it will ever happen